### PR TITLE
bug fixes

### PR DESCRIPTION
**Pull request by bradymiller**
bug fixes

### CODE REVIEW

File: interface/forms/LBF/new.php [hunk 1]
```diff
@@ -201,7 +201,7 @@ function end_row()
 }
 
 if (!$from_trend_form) {
-    $fname = $GLOBALS['OE_SITE_DIR'] . "/LBF/$formname.plugin.php";
+    $fname = $GLOBALS['OE_SITE_DIR'] . "/LBF/" . check_file_dir_name($formname) . ".plugin.php";
```
Comment (stephenwaite) [inline interface/forms/LBF/new.php:204]: jarosiewicz-mr-bean

**Review by stephenwaite — COMMENTED** [review]
simpsons-cheering

a lot of fixes, nice!

File: setup.php [hunk 2]
```diff
@@ -244,12 +251,76 @@ function recursive_writable_directory_test($dir)
...
+    SessionUtil::setupScriptSessionStart();
+    session_regenerate_id(true);
+    $_SESSION = [];
+    CsrfUtils::setupCsrfKey();
+} else {
+    // start up session and check csrf
+    SessionUtil::setupScriptSessionStart();
+    $state = (int)$state;
+    $verifyCsrf = false;
+    if (($state > 0) && ($state < 8)) {
```
Comment (adunsulag) [inline setup.php:287]: Anyway we can put these numbers as constants somewhere?  Makes it hard to follow the logic of what is going on with 0, 8, 3, etc.

Comment (bradymiller) [reply setup.php:287]: i could, but then I'd begin to consider Seppuku :)
i'll do the next best thing, though, and put some comments about what each one is

Comment (adunsulag) [reply setup.php:287]: lol, well let's not get hasty haha.

Comment (bradymiller) [reply setup.php:287]: luckily no sword handy, but there is always amazon prime :)

Comment (bradymiller) [reply setup.php:299]: added more description of the state(s) to the script header comments

**Comment by bradymiller** [comment]
testing well and bringing in
